pkg/core/managers/apis/mesh/mesh_manager.go

```diff
@@ -103,7 +103,7 @@ func (m *meshManager) Delete(ctx context.Context, resource core_model.Resource,
 	}
 	opts := core_store.NewDeleteOptions(fs...)
 	// delete all secrets
-	if err := m.otherManagers.DeleteAll(ctx, &system.SecretResourceList{}, core_store.DeleteAllByMesh(opts.Mesh)); err != nil {
+	if err := m.otherManagers.DeleteAll(ctx, &system.SecretResourceList{}, core_store.DeleteAllByMesh(opts.Name)); err != nil {
```

Comment:
Interesting, was that a bug? Does opts.Name hold the mesh name now, cause the method you call is `DeleteAllByMesh`

Reply:
Yes, it was a bug. Mesh is empty in a Mesh object to indicate mesh scope